runners/katacoda/index.ts

```diff
@@ -416,4 +375,25 @@ export class Katacoda extends Runner {
         return {terminalId: this.terminalCounter, isRunning: false};
     }
 
+    private getStepsCount(runCommand: RunCommand): number {
+        let returnCount = runCommand.stepIndex == this.stepsCount - 1 ? this.stepsCount : ++this.stepsCount;
+        return returnCount;
+    }
+
+    private pushStep(runCommand: RunCommand, title: string, text: string) {
+        if (this.steps.length == this.stepsCount - 1) {
+            if (runCommand.stepTitle) {
+                this.steps.push({
+                    "title": runCommand.stepTitle,
```

Comment:
When you use the ternary operator to set the title you do not need the if-else part